compiler/src/symboltable.rs

```diff
@@ -954,10 +954,10 @@ impl SymbolTableBuilder {
         // Some checks:
         let containing = table.symbols.contains_key(name);
         if containing {
```

Comment:
I think you can replace this with `if let Some(symbol) = table.symbols.get(name) {`

Reply:
Thank you for your tip :) 
I made refactoring to replace `containing` with `Some(symbol)`
PTAL!